src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -15,36 +18,186 @@ namespace Microsoft.Sbom.Utils;
 /// </summary>
 public static class SPDXToSbomFormatConverterExtensions
 {
-    /// <summary>
-    /// Converts a <see cref="SPDXFile"/> object to a <see cref="SbomFile"/> object.
-    /// </summary>
-    /// <param name="spdxFile"></param>
-    /// <returns></returns>
-    public static SbomFile ToSbomFile(this File spdxFile)
+    public static SbomFile ToSbomFile(this File spdxFile, List<Element> spdx30Elements = null, List<Relationship> relationships = null)
     {
-        var checksums = spdxFile.VerifiedUsing?.Select(c => c.ToSbomChecksum());
-
-        // Not setting LicenseConcluded and LicenseInfoInFiles since the whole SBOM is required to set these values.
         return new SbomFile
         {
-            Checksum = checksums,
-            FileCopyrightText = spdxFile.CopyrightText,
+            Checksum = spdxFile.VerifiedUsing.ToSbomChecksum(),
+            FileCopyrightText = spdxFile.CopyrightText == "NOASSERTION" ? null : spdxFile.CopyrightText,
```

Comment:
Why do we want null instead of no assertion?

Reply:
NOASSERTION is the public facing value we use in SPDX documents. Internally, this is equivalent to a null value. 

In our SBOM generation code, when we go from SBOMFile to SPDXFile we convert null values to NOASSERTION. Similarly, in these extension methods I reversed the logic so that the NOASSERTION values are converted to null values.

Reply:
Slight correction: While the above is still true, it looks like SPDX 2.2 extensions does not follow this format and instead just uses whatever is provided in the spdx element. So for example if NOASSERTION is provided then the value remains as NOASSERTION.

Since the point of adding these extensions is to convert all SPDX data to an internal standard, I will conform with what is already in place for SPDX 2.2 and remove the NOASSERTION to null conversions.